finvizfinance/group/custom.py

```diff
@@ -82,7 +82,7 @@ def screener_view(
         soup = web_scrap(self.url)
         table = soup.find("table", class_="table-light")
         rows = table.findAll("tr")
-        table_header = [i.text for i in rows[0].findAll("td")][1:]
+        table_header = [i.text.replace("\n\n", "") for i in rows[0].findAll("td")][1:]
```

Comment:
I think `text.strip()` will remove all the while space

Reply:
You are correct. I forgot that function existed is more convenient and versatile.

Modifying the code to use `text.strip()` instead.